capabilities.go

```diff
@@ -163,44 +163,31 @@ func (c *Client) GetCapabilities(target string, port int) (response *Capabilitie
 // ExtractPikeOutputsURL extracts the outputs URL from the PIKE capability
 func (c *CapabilitiesPayload) ExtractPikeOutputsURL() string {
 	if c.Pike != nil {
-		return c.Pike.Outputs
+		return *c.Pike.Outputs
 	}
 	return ""
 }
 
 // ExtractPikeInviteURL extracts the invite URL from the PIKE capability
 func (c *CapabilitiesPayload) ExtractPikeInviteURL() string {
 	if c.Pike != nil {
-		return c.Pike.Invite
+		return *c.Pike.Invite
 	}
 	return ""
 }
 
 // parsePikeCapability parses the PIKE capability from the capabilities response
 func parsePikeCapability(response *CapabilitiesResponse) error {
 	if pike, ok := response.Capabilities[BRFCPike].(map[string]interface{}); ok {
-		var (
-			invite, outputs string
-			errMsgs         []string
-		)
+		response.Pike = &PikeCapability{}
 
 		if inviteStr, ok := pike["invite"].(string); ok {
-			invite = inviteStr
+			response.Pike.Invite = &inviteStr
 		}
 
+		// TODO: make outputs required
```

Comment:
Please add more context to this comment - like: "... when we implement the PIKE transactions"

Reply:
Done